cmd/config/get/get.go

```diff
@@ -54,20 +55,26 @@ func run(_ *cobra.Command, argv []string) {
 }
 
 func PrintConfig(arg string) error {
-	// Load the configuration file:
-	cfg, err := config.Load()
-	if err != nil {
-		return fmt.Errorf("Failed to load config file: %v", err)
+	// The following variables are not stored in the configuration
+	// and can skip loading configuration:
+	skipConfigLoadMap := map[string]bool{
+		"keyrings": true,
 	}
 
-	// If the configuration file doesn't exist yet assume that all the configuration settings
-	// are empty:
-	if cfg == nil {
-		loc, err := config.Location()
+	cfg := &config.Config{}
+	var err error
+	if !skipConfigLoadMap[arg] {
+		// Load the configuration:
+		cfg, err = config.Load()
 		if err != nil {
-			return fmt.Errorf("Failed to find config file location: %v", err)
+			return fmt.Errorf("can't load config: %v", err)
```

Comment:
@robpblake suggested to show the config file location in the error message (https://github.com/openshift/rosa/pull/1780#discussion_r1505723401). Should we keep it that way?

Reply:
`config.Load() -> loadFromFile` will return the file path if it fails to load from a file. I am removing cases of the word "file" where errors can come from the keyring OR the file now.